automation/project/Settings.scala

```diff
@@ -15,7 +15,8 @@ object Settings {
     "-deprecation",
     "-feature",
     "-encoding", "utf8",
-    "-target:jvm-1.8"
+    "-target:jvm-1.8",
+    "-Xmax-classfile-name", "100"
```

Comment:
Just checking that this was an intentional addition.

Reply:
this is a fix brian hasn't merged yet. good find